packages/backend/src/controllers/announcements/announcements.controller.ts

```diff
@@ -80,6 +90,11 @@ export class AnnouncementController {
     );
   }
 
+  /**
+   * Get all Announcements for the authenticated user's house
```

Comment:
I almost feel like these endpoint comments are redundant with the swagger docs annotations below.

Reply:
Yeah, I somewhat agree tbh. Much of the code is self-documenting. I just added this to have docs for all the functions that we added. What do you think we should do?

Reply:
personally, i think that for any endpoints that make use of the swagger annotations we shouldn't add extra jsdoc comments since that's redundant to have it in two places and also has the potential to lead to conflicting documentation which is bad obvs, like i think they won't mark us down because swagger is the documentation that we added for these endpoints

Reply:
Agreed, I'll remove the jsdocs for the controllers.